source/setup.py

```diff
@@ -237,7 +237,8 @@ def _genManifestTemplate(shouldHaveUIAccess: bool) -> tuple[int, int, bytes]:
 		("louis/tables",glob("louis/tables/*")),
 		("COMRegistrationFixes", glob("COMRegistrationFixes/*.reg")),
 		(".", glob("../miscDeps/python/*.dll")),
-		(".", ['message.html' ])
+		(".", ['message.html']),
+		(".", [os.path.join(sys.base_prefix, "python3.dll")])

```

Comment:
added comma to minimize future diffs

```suggestion
		(".", [os.path.join(sys.base_prefix, "python3.dll")]),
```